<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgCalProjector</class>
 <widget class="QDialog" name="dlgCalProjector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>547</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calibrate XY Resolution (Projector)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditStatus">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>204</green>
           <blue>204</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>204</green>
           <blue>204</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="toolTip">
      <string>Status</string>
     </property>
     <property name="whatsThis">
      <string>Status - reports if printer is in motion or read for input.</string>
     </property>
     <property name="text">
      <string>Ready...</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButtonDone">
     <property name="toolTip">
      <string>Cancel or Finished</string>
     </property>
     <property name="whatsThis">
      <string>Cancel to exit.</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Calibrate the Projector for the desired XY Pixel Resolution</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="toolTip">
         <string>Instructions</string>
        </property>
        <property name="whatsThis">
         <string>Instructions - For more detailed information visit www.b9creator.com</string>
        </property>
        <property name="text">
         <string>The purpose of Projector Calibration is to ensure the projected image is focused and the projected pixels match the selected XY pixel size.  This is acomplished by projecting a grid pattern onto a printed grid template and adjusting the position, zoom and focus of the projector until the projected grid image matches the paper grid template.

Go to www.b9creator.com/documentation/ to download and print the reference grid patterns.

You must select the XY Pixel Resolution that best suits your needs.  Higher resolution produces a better surface finish but the maximum build volume is lower.

The 3 XY Pixel resolution options are:
   High - 50 micron pixels - Build volume of 51.2 x 38.4 x200 mm (2&quot; x 1.5&quot; x 8&quot;)
   Medium - 75 micron pixels - Build volume of 76.8 x 57.6 x 200 mm (3&quot; x 2.27&quot; x 8&quot;)
   Low - 100 micron pixels - Build volume of 102.4 x 76.8 x 200 mm (4&quot; x 3&quot; x 8&quot;)

Ensure the Projector is connected via VGA or HDMI cable and display is set to &quot;Extended&quot; Mode.
</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Projector Connection Status:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditStatusDisplay">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>204</red>
              <green>204</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="toolTip">
         <string>Status</string>
        </property>
        <property name="whatsThis">
         <string>Status - reports if printer is in motion or read for input.</string>
        </property>
        <property name="text">
         <string>Ready...</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Step 1 - Select the desired XY Pixel Resolution:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="comboBoxXPPixelSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>XY Pixel Size</string>
        </property>
        <property name="whatsThis">
         <string>Set the desired XY Pixel Size for this calibration.</string>
        </property>
        <item>
         <property name="text">
          <string>50 microns</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>75 microns</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100 microns</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxStep1">
        <property name="toolTip">
         <string>Step 1</string>
        </property>
        <property name="whatsThis">
         <string>Step 1 of 6</string>
        </property>
        <property name="text">
         <string>Step 2 - Ensure the VAT is in place and the Build Table and Sweeper are removed.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxStep2">
        <property name="toolTip">
         <string>Step 2</string>
        </property>
        <property name="whatsThis">
         <string>Step 2 of 6</string>
        </property>
        <property name="text">
         <string>Step 3 - Loosen the 4 screws on the back of the printer that hold projector in place.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Note:  The next step may take a few minutes as the projector warms up. </string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>(If using a single video output and spliter setup then see the web site for additional documentation.)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QPushButton" name="pushButtonStep3">
        <property name="toolTip">
         <string>Step 3</string>
        </property>
        <property name="whatsThis">
         <string>Step 3 of 6</string>
        </property>
        <property name="text">
         <string>Step 4 - Click to power up the projector, reset the printer and open the VAT.</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxStep4">
        <property name="toolTip">
         <string>Step 4</string>
        </property>
        <property name="whatsThis">
         <string>Step 4 of 6</string>
        </property>
        <property name="text">
         <string>Step 5 - Adjust the Zoom, Focus and Position of the projector until the projector grid matches the paper template grid.</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="3">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>(The paper template should be placed printed side up on the bottom of the vat and aligned with the projected grid manually.)</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxStep5">
        <property name="toolTip">
         <string>Step 5</string>
        </property>
        <property name="whatsThis">
         <string>Step5 of 6</string>
        </property>
        <property name="text">
         <string>Step 6 - Tighten the 4 projector screws and double check the projected image and focus.</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonStep6">
        <property name="toolTip">
         <string>Step 6</string>
        </property>
        <property name="whatsThis">
         <string>Step 6 of 6</string>
        </property>
        <property name="text">
         <string>Step 7 - Click here to turn off the projector.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxStep1</sender>
   <signal>clicked(bool)</signal>
   <receiver>dlgCalProjector</receiver>
   <slot>onStep1(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStep2</sender>
   <signal>clicked(bool)</signal>
   <receiver>dlgCalProjector</receiver>
   <slot>onStep2(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStep3</sender>
   <signal>clicked()</signal>
   <receiver>dlgCalProjector</receiver>
   <slot>onStep3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStep4</sender>
   <signal>clicked(bool)</signal>
   <receiver>dlgCalProjector</receiver>
   <slot>onStep4(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStep5</sender>
   <signal>clicked(bool)</signal>
   <receiver>dlgCalProjector</receiver>
   <slot>onStep5(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStep6</sender>
   <signal>clicked()</signal>
   <receiver>dlgCalProjector</receiver>
   <slot>onStep6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>dlgCalProjector</receiver>
   <slot>done()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStep1(bool)</slot>
  <slot>onStep2(bool)</slot>
  <slot>onStep3()</slot>
  <slot>onStep4(bool)</slot>
  <slot>onStep5(bool)</slot>
  <slot>onStep6()</slot>
  <slot>done()</slot>
 </slots>
</ui>
